Tasks/AzureResourceManagerTemplateDeploymentV3/operations/ParameterParser.ts

```diff
@@ -0,0 +1,154 @@
+// This class manages the powershell parameters format
```

Comment:
this file is lifted as such from previous version. right?

Reply:
Yes, all the files are from previous version. Please have a look at this https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks/AzureResourceGroupDeploymentV2